src/devices/Dhtxx/DhtSensor.cs

```diff
@@ -0,0 +1,307 @@
+        /// </remarks>
+        public double TemperatureInCelsius
+        {
+            get
+            {
+                if (_dhtType == DhtType.Dht11)
+                    return GetTempDht11();
+                else
+                    return GetTempDht22();
+            }
+        }
+
+        /// <summary>
+        /// Get the last read temperature in Farenheit
+        /// </summary>
+        /// <remarks>
+        /// If last read was not successfull, it returns double.MaxValue
+        /// </remarks>
+        public double TemperatureInFarenheit
+        {
+            get
+            {
+                // °F = 9/5 °C + 32
+                if (LastReadSuccessful)
+                    return (9.0 / 5.0 * TemperatureInCelsius + 32);
+                else
+                    return Double.MaxValue;
+            }
+        }
+
+        /// <summary>
+        /// Get the temperature in Celsius
+        /// </summary>
+        /// <param name="temperatureInCelsius">The temperature in Celsius</param>
+        /// <returns>Returns <c>true</c> if the read is successful</returns>
+        /// <remarks>
+        /// If last read was not successfull, it returns double.MaxValue
+        /// </remarks>
+        public bool TryGetTemperatureInCelsius(out double temperatureInCelsius)
+        {
+            var ret = ReadData();
+            temperatureInCelsius = TemperatureInCelsius;
+            return ret;
+        }
+
+        /// <summary>
+        /// Get the temperature in Farenheit
+        /// </summary>
+        /// <param name="temperatureInFarenheit">The temperature in Farenheit</param>
+        /// <returns>Returns <c>true</c> if the read is successful</returns>
+        /// <remarks>
+        /// If last read was not successfull, it returns double.MaxValue
+        /// </remarks>
+        public bool TryGetTemperatureInFarenheit(out double temperatureInFarenheit)
+        {
+            var ret = ReadData();
+            temperatureInFarenheit = TemperatureInFarenheit;
+            return ret;
+        }
+
+        /// <summary>
+        /// Get the last read humidity
+        /// </summary>
+        /// <remarks>
+        /// If last read was not successfull, it returns double.MaxValue
+        /// </remarks>
+        public double Humidity
+        {
+            get
+            {
+                if (_dhtType == DhtType.Dht11)
+                    return GetHumidityDht11();
+                else
+                    return GetHumidityDht22();
+            }
+        }
+
+        /// <summary>
+        /// Get the relative humidity in the air
+        /// </summary>
+        /// <param name="humidity">The percentage of humidity in the air</param>
+        /// <returns>Returns <c>true</c> if the read is successful</returns>
+        /// <remarks>
+        /// If last read was not successfull, it returns double.MaxValue
+        /// </remarks>
+        public bool TryGetHumidity(out double humidity)
```

Comment:
Should this be "TryGetRelativeHumidity(out double relativeHumidity)" since it's a percentage.  same for all other humidty parameters elsewhere